odoo/addons/base/ir/ir_mail_server.py

```diff
@@ -464,6 +465,70 @@ def send_email(self, message, mail_server_id=None, smtp_server=None, smtp_port=N
             raise MailDeliveryException(_("Mail Delivery Failed"), msg)
         return message_id
 
+    @api.multi
+    def _get_canonical_from(self, original_from):
```

Comment:
Any idea for an updated name for `_get_canonical_from()`? Does `_get_rewritten_from()` sound awkward?